### ISSUE TITLE
Error due to missing `is_zero` arg when saving LR scheduler

### ISSUE THREAD

**Issue by Lauler** [huggingface/nanotron#252]
This recent commit https://github.com/huggingface/nanotron/commit/51bd072a59b37eb73f980207b5dd831243332969 and pull request https://github.com/huggingface/nanotron/pull/230 which changes how LR schedulers are saved forgot to add the `is_zero` arg and set it to `config.optimizer.zero_stage` in `src/nanotron/serialize/main.py`:

https://github.com/huggingface/nanotron/blob/cfcdeae812d4c7210990932ebcce5f672084408d/src/nanotron/serialize/main.py#L106-L110

This arg is expected to be passed:

https://github.com/huggingface/nanotron/blob/cfcdeae812d4c7210990932ebcce5f672084408d/src/nanotron/serialize/optimizer.py#L109-L114

This causes training to crash when LR scheduler is saved.

@TJ-Solergibert @NouamaneTazi

**Comment by sankexin** [comment]
[nanotron/src/nanotron/serialize/main.py]
```
save_lr_scheduler(
                lr_scheduler=lr_scheduler,
                is_zero=True,
                parallel_context=parallel_context,
                root_folder=root_folder,
            )
```

**Comment by Lauler** [comment]
This was fixed in https://github.com/huggingface/nanotron/commit/521f759811b243d42d443b1cecca79913ff3e7dc
